ToNote/Views/MainView.xaml.cs

```diff
@@ -11,5 +11,23 @@ public MainView()
         {
             InitializeComponent();
         }
+
+        protected void Minimize_Window(object sender, RoutedEventArgs e)
+        {
+            if (this.WindowState == WindowState.Normal)
+                this.WindowState = WindowState.Minimized;
+        }
+
+        protected void Maximize_Window(object sender, RoutedEventArgs e)
+        {
+            if (this.WindowState == WindowState.Normal)
```

Comment:
Your maximize/minimize behavior is a bit broken. 
First, with default impl you can use the middle button to both maximize and minimize the window. With the current impl, you can use it to maximize the window, but not to reduce it back to the initial size.
Second, minimize button does not work if the window is maximized.
Third, with current impl, when the window is maximized, it hides the taskbar. This would be fine as a toggleable option in the future, but should not be the default behavior.